src/web/components/LayoutSlotBottom.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { css } from 'emotion';
+import { useApi } from '@root/src/web/components/lib/api';
+
+const wrapperMargins = css`
+    margin: 18px 0;
```

Comment:
That's true, we don't need a top margin here but I added it anyway just to be on the safe side in case the article body ends with an element which doesn't have a bottom margin for whatever reason. Or if we move this component somewhere else where such spacing is not there.